solc/CommandLineParser.cpp

```diff
@@ -578,6 +579,11 @@ General Information)").c_str(),
 			"Supported Inputs is the output of the --" + g_strStandardJSON + " or the one produced by "
 			"--" + g_strCombinedJson + " " + g_strAst + "," + g_strCompactJSON).c_str()
 		)
+		(
+			g_strImportAssemblyJson.c_str(),
+			("Import assembly to be used, assumes input holds assembly in JSON format. "
+			 "Supported Input is the output of the --" + g_strAsmJson + ".").c_str()
```

Comment:
This has to be clarified a bit more, I think.

Reply:
Maybe, what should be clarified a bit more?